### PR TITLE
Update core to 1.147

### PR DESCRIPTION
**Pull request by nicodh**

### CODE REVIEW

**Review by iequidoo — APPROVED** [review]

**Comment by nicodh** [comment]
Seems we had to adapt settings first. See https://github.com/deltachat/deltachat-desktop/issues/4124

**Comment by nicodh** [comment]
Step 1 of #4124 is included here now:

image

**Review by r10s — COMMENTED** [review]
i did a quick functionality check and could manage to connect via `ss://YWVzLTEyOC1nY206c2hhZG93c29ja3M=@156.146.62.163:443#Switzerland%20#1270%20/%20OutlineKeys.com` - nice!

for the UI: as this is subject to change anyways and will do to a dedicated dialog, it is good enough for now.

for the strings:  they are available already translated, no need to add them to "untranslated".
for the "placeholder", i'd strike that for now, that seems more confusing and cluttering than helpful - ppl will have a proxy URL from some sources and needs and will paste them here - and not the other way round. also, already now it is incomplete as `ss` is missing.
if we think, a placeholder is  useful, let's start with that on android, where we move ahead with proxy settings and collect more experiences.


i cannot say much to the rest of the code

**Comment by nicodh** [comment]
> for the strings: they are available already translated, no need to add them to "untranslated".

where can I check what is already translated?
Do I need to become member in the transifex DeltaChat project?

**Comment by nicodh** [comment]
> for the "placeholder", i'd strike that for now, that seems more confusing and cluttering than helpful -

I agree  - at least as long the placeholder has the same styling than the entered text

**Comment by r10s** [comment]
note that there is 1.147.1 meanwhile, 1.147.0 contained a bug that resulted in some incompatibilities with existing releases, see https://github.com/deltachat/deltachat-core-rust/issues/6038

File: packages/frontend/src/components/LoginForm.tsx [hunk 1]
```diff
@@ -311,7 +311,12 @@ export default function LoginForm({ credentials, setCredentials }: LoginProps) {
             />
             {proxy_enabled === '1' && (
               <>
-                <p className='text'>{tx('login_proxy_experimental_warning')}</p>
+                <p className='text'>
+                  Proxy support is currently experimental. Please use at your
+                  own risk. If you type in an address in the e-mail field, there
+                  there will be DNS lookup that won't get tunneled through
```
Comment (iequidoo) [inline packages/frontend/src/components/LoginForm.tsx:317]: ~~there~~ will be **a** DNS lookup that won't get tunneled through

**Pull request merged**